### PR TITLE
📖 Document special handling of system infrastructure objects

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR updates the PoC outline to recognize that certain individual objects are part of the nature of some other object or system and should be excepted from downsync.

## Related issue(s)

Fixes #

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
The force-push to 6943d436 is a rebase onto `main`.

File: docs/content/en/docs/Coding Milestones/PoC2023q1/outline.md [hunk 1]
```diff
@@ -405,6 +414,25 @@ this category.
...
+tolerated in the center and do not propagate toward the edge.  Here is
+an initial list of system infrastructure objects:
+
+- `Secret` objects whose _type_ is
+  `kubernetes.io/service-account-token` (these are automatically
+  created to support a `ServiceAccount` in some circumstances) or
+  `bootstrap.kubernetes.io/token`;
+- `ConfigMap` objects named `kube-root-ca.crt`;
+- `ServiceAcount` objects named `default` (these are automatically
+  created created as a consequence of a namespace being created).
```
Comment (waltforme) [inline docs/content/en/docs/Coding Milestones/PoC2023q1/outline.md:434]: Automatically created twice :)

Comment (MikeSpreitzer) [reply docs/content/en/docs/Coding Milestones/PoC2023q1/outline.md:434]: Yes. Two different kinds of objects here are automatically created.

Comment (MikeSpreitzer) [reply docs/content/en/docs/Coding Milestones/PoC2023q1/outline.md]: Fixed in latest push

**Pull request merged**